SwiftWisdom/Core/StandardLibrary/RawRepresentable/RawRepresentable.swift

```diff
@@ -9,6 +9,7 @@
 import Foundation
 
 extension RawRepresentable where RawValue: BinaryInteger {
+    // TODO: May be deprecated by CaseIterable protocol and allCases computed property.
```

Comment:
I think that as a team, we should debate whether to deprecate *now* on a "major" release (well, as major as we get), or if we want to wait for some future point release.  I assume that it would be something along the lines of....

```swift
@available(*, deprecated, message: "Conform to CaseIterable and use allCases instead")
```

Reply:
I agree, it would be good to have a brief discussion about this.

Reply:
Yeah, I think it's worth adding the deprecated flag as Bob mentioned. `CaseIterable` does it better than we do it.